Source/Fuse.ImageTools/Tests/ImageTools.Test.uno

```diff
@@ -10,49 +10,31 @@ namespace Fuse.ImageTools.Test
 	{
 		private Object _image = null;
 
-
 		[Test]
+		[Ignore("Only supported on dotnet", "!DOTNET")]
 		public void CreatingAnImageSucceeds()
 		{
-			if defined(!DOTNET) return;
-
 			Fuse.ImageTools.ImageTools image = new Fuse.ImageTools.ImageTools();
 			using (var root = new TestRootPanel()){
 				var future = ImageTools.ImageFromBase64("iVBORw0KGgoAAAANSUhEUgAAAA8AAAAPAQMAAAABGAcJAAAABlBMVEX9//wAAQATpOzaAAAAH0l" + "EQVQI12MAAoMHIFLAAYSEwIiJgYGZASrI38AAAwBamgM5VF7xgwAAAABJRU5ErkJggg==");
 				future.Then(Print, Fail);
-				root.StepFrame();
-				root.StepFrame();
-				root.StepFrame();
-				root.StepFrame();
-
 				Assert.IsFalse(_image == null);
```

Comment:
This just proves that `void Fail(Exception e)` hasn't been called (potentially... yet)